### PR TITLE
Download using objectURL of a blob

### PR DESCRIPTION
**Pull request by kba**
The data-URL technique only works for reasonably short strings, this is a more robust implementation. Should work for transformations from URL and upload.

### CODE REVIEW

File: web/ocr-fileformat.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 /* globals $ */
+/* globals Blob */
+/* global Prism */
```
Comment (stweil) [inline web/ocr-fileformat.js]: Is there a difference between `globals` (as in line 2) and `global` (as in line 3) here?

Comment (kba) [reply web/ocr-fileformat.js]: No.

File: Dockerfile [hunk 2]
```diff
@@ -9,6 +9,7 @@ RUN apk add --no-cache openjdk8-jre php7 php7-json php7-openssl python py-lxml g
     && update-ca-certificates \
     && make install \
     && cp docker.config.php web/config.local.php \
+    && sed -i '/^upload_max_filesize/ s/=.*$/= 100M' \
```
Comment (zuphilip) [inline Dockerfile]: There seems to be a problem with the automated Docker build on docker hub: https://hub.docker.com/r/ubma/ocr-fileformat/builds/bvafgbah9zqysoblyvupmeg/

Comment (zuphilip) [reply Dockerfile]: Ping @kba 🔥

**Comment by stweil** [comment]
I have committed a fix for `Dockerfile` which hopefully make it work again.

File: Dockerfile (hunk 2 above)
Comment (kba) [reply Dockerfile]: Fixed. Since the image sizes don't differ now, I'll sumit my dev Dockerfile later.

**Pull request merged**